package/gluon-core/luasrc/lib/gluon/upgrade/022-wireless-roles

```diff
@@ -66,6 +67,23 @@ local function migrate_wifi_to_roles(band)
 				end
 			end
 		end
+
+		uci:section('gluon', 'wireless_radio', radio_name, {
+		})
+
+		local txpower = uci:get('wireless', radio_name, 'txpower')
+		if txpower ~= nil then
+			uci:set('gluon', radio_name, 'txpower', txpower)
+		end
+
+		-- remove in 2028 or when upgrades from v2025 are unsupported
```

Comment:
i'm not sure i understand this sentence. and shouldn't we make it more obvious to mark it for later removal with a more eye-catching or searchable keyword?